### DISCUSSION TITLE
Show extended entity fields in admin ui

### DISCUSSION THREAD

**Discussion by SubhrajitDG** [medusajs/medusa#8814] [Q&A]
Hi, i recently started exploring medusa and i want to have more properties on my products. So i know that i can extend the product entity but i also want fields for those properties to be available on admin ui product creation. Is it possible and how to do so?

I tried adding the customFields in medusa config but it doesn't work
`module.exports = {
  projectConfig,
  plugins,
  customFields: {
    product: [
      {
        name: "some_property",
        type: "string",
      },
    ],
  },
  modules,
  featureFlags,
};`

**Comment by olivermrbl** [comment]
There is no such thing as `customFields` in our config. Where did you find this?

**Reply by SubhrajitDG** [reply]
hi @olivermrbl , this answer posted by you on another discussion.
https://github.com/medusajs/medusa/discussions/1947#discussioncomment-4730840

Nonetheless please let me know if i will be able to achieve what i am trying to do.
